### PR TITLE
[iOS] Update the Sample App for iOS

### PR DESCRIPTION
**Pull request by minggangw**
This patch does:

1. Updates the MSQASignIn framework that includes the recent changes, which are:

   - Support the telemetry functionality.
   - Support the logging functionality.
   - Support the given name/surname/email for MSA account.

3. Updates the Sample app accordingly per the changes above.

### CODE REVIEW

**Comment by minggangw** [comment]
Only the file, `demos/iOS/SampleAppiOS/SampleMainViewController.m`, gets updated for the sample app itself, for other changes are introduced from the binary framework located at `demos/iOS/MSQASignIn.framework/`

File: demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQALogger.h [hunk 1]
```diff
@@ -0,0 +1,81 @@
+//------------------------------------------------------------------------------
```
Comment (chbhatt) [inline demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQALogger.h:1]: Shouldn't MSQALogger.h be public header?

File: demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQAAccountData_Private.h [hunk 2]
```diff
@@ -39,7 +39,13 @@ NS_ASSUME_NONNULL_BEGIN
                          idToken:(nullable NSString *)idToken
                      accessToken:(nullable NSString *)accessToken;
 
-- (void)setPhoto:(NSString *)photo;
+- (void)setBase64Photo:(NSString *)base64Photo;
```
Comment (chbhatt) [inline demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQAAccountData_Private.h:42]: Do we need to put the PrivateHeaders into framework?

**Review by chbhatt — CHANGES_REQUESTED** [review]

File: demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQALogger.h (hunk 1 above)
Comment (minggangw) [reply demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQALogger.h]: has moved to public

File: demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQAAccountData_Private.h (hunk 2 above)
Comment (minggangw) [reply demos/iOS/MSQASignIn.framework/PrivateHeaders/MSQAAccountData_Private.h]: Has moved all "xxx_Private.h" headers out of the framework

**Review by chbhatt — APPROVED** [review]

**Pull request merged**